assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/inputs/styles.scss

```diff
@@ -0,0 +1,13 @@
+$input-number: 'input[type=' number ']';
+
+.ee-input {
+	&__price-field {
+		&--has-error {
+			// this was used to increase specificity, more info: https://sass-lang.com/documentation/at-rules/at-root
+			@at-root #{$input-number}#{&} {
+				background-color: rgb(255, 255, 255);
+				border-color: rgb(255, 77, 79);
```

Comment:
@tn3rb need an advice here on what are the most appropriate variables

Reply:
- `--ee-background-color`
- `--ee-border-color`
- `--ee-default-text-color` for text color

i tried to have most variables match the selectors, so in most cases you can simply append `--ee-` to the selector name (but not always)

Reply:
thx, I've applied high contrast, but seem a bit too much

![image](https://user-images.githubusercontent.com/1477453/78239139-0990b500-74e6-11ea-8c71-03510d6c78b6.png)

Reply:
low contrast is okish, but a bit too soft:

![image](https://user-images.githubusercontent.com/1477453/78239319-3d6bda80-74e6-11ea-9278-4fe7b8c9d92e.png)

Reply:
if it's not ok, will adjust in followup PR related to the same issue